flask_websub/subscriber/blueprint.py

```diff
@@ -58,7 +58,13 @@ def confirm_subscription(callback_id):
         else:  # unsubscribe
             del subscriber.storage[callback_id]
         subscriber.call_all('success_handlers', topic_url, callback_id, mode)
-        return get_query_arg('hub.challenge'), 200
+
+        challenge = get_query_arg('hub.challenge')
+        mimetype = 'application/octet-stream'
+        response = Response(challenge, status=200, mimetype=mimetype)
+        response.headers['Content-Security-Policy'] = "default-src 'none'"
```

Comment:
I don't think this header field is required. I would not include it.

Reply:
Ok, I'm going to err on the side of caution and keep the CSP header. If it causes issues in someone's setup, it's easy to remove at that point.

Reply:
Thanks for bringing this to my attention and your feedback!

Reply:
Thanks to you as well.